2023-10-26 12:25:32,356 (WARNING) - djongo.sql2mongo.query - Not implemented alter command for SQL ALTER TABLE "api_simpleplayer" ALTER COLUMN "id" TYPE int
2023-10-26 18:25:46,557 (WARNING) - djongo.sql2mongo.query - Not implemented alter command for SQL ALTER TABLE "api_classification" ALTER COLUMN "coef" TYPE string
2023-10-29 15:21:38,222 (WARNING) - djongo.sql2mongo.query - Not implemented alter command for SQL ALTER TABLE "api_clasificacion" ALTER COLUMN "id" TYPE int
2023-10-29 15:22:15,808 (WARNING) - djongo.sql2mongo.query - Not implemented alter command for SQL ALTER TABLE "api_clasificacion" ALTER COLUMN "id" TYPE int
2023-10-29 15:22:23,217 (WARNING) - djongo.sql2mongo.query - Not implemented alter command for SQL ALTER TABLE "api_clasificacion" ALTER COLUMN "id" TYPE int
2023-10-29 15:22:31,767 (WARNING) - djongo.sql2mongo.query - Not implemented alter command for SQL ALTER TABLE "api_clasificacion" ALTER COLUMN "id" TYPE int
2023-10-29 15:32:54,230 (WARNING) - djongo.sql2mongo.query - Not implemented alter command for SQL ALTER TABLE "api_clasificacion" ALTER COLUMN "id" TYPE int
2023-10-29 15:34:03,193 (WARNING) - djongo.sql2mongo.query - Not implemented alter command for SQL ALTER TABLE "api_clasificacion" ALTER COLUMN "id" TYPE int
2023-11-01 18:15:06,184 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-01 18:15:12,079 (WARNING) - django.request - Not Found: /favicon.ico
2023-11-01 18:15:12,090 (WARNING) - django.request - Not Found: /apple-touch-icon-precomposed.png
2023-11-01 18:15:12,098 (WARNING) - django.request - Not Found: /apple-touch-icon.png
2023-11-01 18:25:59,750 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/admin.py changed, reloading.
2023-11-01 18:26:00,173 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-01 18:27:36,460 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/admin.py changed, reloading.
2023-11-01 18:27:36,988 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-01 18:33:29,433 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/admin.py changed, reloading.
2023-11-01 18:33:29,788 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-01 18:33:41,047 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/admin.py changed, reloading.
2023-11-01 18:33:41,365 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-01 18:34:05,018 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/admin.py changed, reloading.
2023-11-01 18:34:05,777 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-01 18:34:08,947 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/admin.py changed, reloading.
2023-11-01 18:34:09,138 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-01 18:34:54,799 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/admin.py changed, reloading.
2023-11-01 18:34:55,154 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-01 18:36:32,384 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/admin.py changed, reloading.
2023-11-01 18:36:32,991 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-01 18:36:39,197 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/admin.py changed, reloading.
2023-11-01 18:36:39,423 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-01 18:40:16,640 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/admin.py changed, reloading.
2023-11-01 18:40:17,394 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-01 18:40:30,854 (ERROR) - django.request - Internal Server Error: /admin/api/clasificacion/
Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/models/options.py", line 669, in get_field
    return self.fields_map[field_name]
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^
KeyError: "{'id': 2107, 'group': 1, 'group_name': None, 'conference': 0, 'team': 'Real Madrid', 'points': 28, 'wins': 9, 'draws': 1, 'losses': 1, 'shield': 'https://t.resfu.com/img_data/escudos/medium/2107.jpg?size=120x&lossy=1&v=36', 'cflag': 'https://t.resfu.com/media/img/flags/st3/large/es.png?size=100x65c&lossy=1&v=36', 'basealias': 'real-madrid', 'gf': 23, 'ga': 8, 'avg': 15, 'matchs_coef': '', 'points_coef': '', 'coef': '', 'coefficients': None, 'mark': '1', 'class_mark': 'campeon', 'round': 12, 'pos': 1, 'countrycode': 'ES', 'abbr': 'RMA', 'form': 'wwwdw', 'direction': ''}"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 333, in label_for_field
    field = _get_non_gfk_field(model._meta, name)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 302, in _get_non_gfk_field
    field = opts.get_field(name)
            ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/models/options.py", line 671, in get_field
    raise FieldDoesNotExist(
django.core.exceptions.FieldDoesNotExist: Clasificacion has no field named '{'id': 2107, 'group': 1, 'group_name': None, 'conference': 0, 'team': 'Real Madrid', 'points': 28, 'wins': 9, 'draws': 1, 'losses': 1, 'shield': 'https://t.resfu.com/img_data/escudos/medium/2107.jpg?size=120x&lossy=1&v=36', 'cflag': 'https://t.resfu.com/media/img/flags/st3/large/es.png?size=100x65c&lossy=1&v=36', 'basealias': 'real-madrid', 'gf': 23, 'ga': 8, 'avg': 15, 'matchs_coef': '', 'points_coef': '', 'coef': '', 'coefficients': None, 'mark': '1', 'class_mark': 'campeon', 'round': 12, 'pos': 1, 'countrycode': 'ES', 'abbr': 'RMA', 'form': 'wwwdw', 'direction': ''}'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/templatetags/base.py", line 45, in render
    return super().render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/library.py", line 258, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/templatetags/admin_list.py", line 326, in result_list
    headers = list(result_headers(cl))
              ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/templatetags/admin_list.py", line 90, in result_headers
    text, attr = label_for_field(
                 ^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 361, in label_for_field
    raise AttributeError(message)
AttributeError: Unable to lookup '{'id': 2107, 'group': 1, 'group_name': None, 'conference': 0, 'team': 'Real Madrid', 'points': 28, 'wins': 9, 'draws': 1, 'losses': 1, 'shield': 'https://t.resfu.com/img_data/escudos/medium/2107.jpg?size=120x&lossy=1&v=36', 'cflag': 'https://t.resfu.com/media/img/flags/st3/large/es.png?size=100x65c&lossy=1&v=36', 'basealias': 'real-madrid', 'gf': 23, 'ga': 8, 'avg': 15, 'matchs_coef': '', 'points_coef': '', 'coef': '', 'coefficients': None, 'mark': '1', 'class_mark': 'campeon', 'round': 12, 'pos': 1, 'countrycode': 'ES', 'abbr': 'RMA', 'form': 'wwwdw', 'direction': ''}' on Clasificacion or ClassificationAdmin
2023-11-01 18:42:43,047 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/admin.py changed, reloading.
2023-11-01 18:42:43,396 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-01 18:42:51,269 (WARNING) - django.request - Not Found: /admin/api/clasificacion/
2023-11-01 18:43:08,361 (WARNING) - django.request - Not Found: /admin/api/clasificacion/
2023-11-01 18:43:15,903 (ERROR) - django.request - Internal Server Error: /admin/api/table/
Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/models/options.py", line 669, in get_field
    return self.fields_map[field_name]
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^
KeyError: "{'id': 2107, 'group': 1, 'group_name': None, 'conference': 0, 'team': 'Real Madrid', 'points': 28, 'wins': 9, 'draws': 1, 'losses': 1, 'shield': 'https://t.resfu.com/img_data/escudos/medium/2107.jpg?size=120x&lossy=1&v=36', 'cflag': 'https://t.resfu.com/media/img/flags/st3/large/es.png?size=100x65c&lossy=1&v=36', 'basealias': 'real-madrid', 'gf': 23, 'ga': 8, 'avg': 15, 'matchs_coef': '', 'points_coef': '', 'coef': '', 'coefficients': None, 'mark': '1', 'class_mark': 'campeon', 'round': 12, 'pos': 1, 'countrycode': 'ES', 'abbr': 'RMA', 'form': 'wwwdw', 'direction': ''}"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 333, in label_for_field
    field = _get_non_gfk_field(model._meta, name)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 302, in _get_non_gfk_field
    field = opts.get_field(name)
            ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/models/options.py", line 671, in get_field
    raise FieldDoesNotExist(
django.core.exceptions.FieldDoesNotExist: Table has no field named '{'id': 2107, 'group': 1, 'group_name': None, 'conference': 0, 'team': 'Real Madrid', 'points': 28, 'wins': 9, 'draws': 1, 'losses': 1, 'shield': 'https://t.resfu.com/img_data/escudos/medium/2107.jpg?size=120x&lossy=1&v=36', 'cflag': 'https://t.resfu.com/media/img/flags/st3/large/es.png?size=100x65c&lossy=1&v=36', 'basealias': 'real-madrid', 'gf': 23, 'ga': 8, 'avg': 15, 'matchs_coef': '', 'points_coef': '', 'coef': '', 'coefficients': None, 'mark': '1', 'class_mark': 'campeon', 'round': 12, 'pos': 1, 'countrycode': 'ES', 'abbr': 'RMA', 'form': 'wwwdw', 'direction': ''}'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/templatetags/base.py", line 45, in render
    return super().render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/library.py", line 258, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/templatetags/admin_list.py", line 326, in result_list
    headers = list(result_headers(cl))
              ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/templatetags/admin_list.py", line 90, in result_headers
    text, attr = label_for_field(
                 ^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 361, in label_for_field
    raise AttributeError(message)
AttributeError: Unable to lookup '{'id': 2107, 'group': 1, 'group_name': None, 'conference': 0, 'team': 'Real Madrid', 'points': 28, 'wins': 9, 'draws': 1, 'losses': 1, 'shield': 'https://t.resfu.com/img_data/escudos/medium/2107.jpg?size=120x&lossy=1&v=36', 'cflag': 'https://t.resfu.com/media/img/flags/st3/large/es.png?size=100x65c&lossy=1&v=36', 'basealias': 'real-madrid', 'gf': 23, 'ga': 8, 'avg': 15, 'matchs_coef': '', 'points_coef': '', 'coef': '', 'coefficients': None, 'mark': '1', 'class_mark': 'campeon', 'round': 12, 'pos': 1, 'countrycode': 'ES', 'abbr': 'RMA', 'form': 'wwwdw', 'direction': ''}' on Table or ClassificationAdmin
2023-11-01 18:50:17,941 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/admin.py changed, reloading.
2023-11-01 18:50:18,355 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-01 18:50:23,962 (ERROR) - django.request - Internal Server Error: /admin/api/table/
Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/options.py", line 686, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/utils/decorators.py", line 134, in _wrapped_view
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/views/decorators/cache.py", line 62, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/sites.py", line 242, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/utils/decorators.py", line 134, in _wrapped_view
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/options.py", line 1932, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/options.py", line 827, in get_changelist_instance
    list_display = self.get_list_display(request)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/pypro/djongofootball/api/admin.py", line 42, in get_list_display
    fields_to_display = [str(field) for field in serializer]
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/pypro/djongofootball/api/admin.py", line 42, in <listcomp>
    fields_to_display = [str(field) for field in serializer]
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/rest_framework/serializers.py", line 538, in __iter__
    yield self[field.field_name]
          ~~~~^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/rest_framework/serializers.py", line 542, in __getitem__
    value = self.data.get(key)
            ^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/rest_framework/serializers.py", line 555, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/rest_framework/serializers.py", line 249, in data
    raise AssertionError(msg)
AssertionError: When a serializer is passed a `data` keyword argument you must call `.is_valid()` before attempting to access the serialized `.data` representation.
You should either call `.is_valid()` first, or access `.initial_data` instead.
2023-11-01 18:50:52,423 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/admin.py changed, reloading.
2023-11-01 18:50:52,780 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-01 18:50:57,488 (ERROR) - django.request - Internal Server Error: /admin/api/table/
Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/models/options.py", line 669, in get_field
    return self.fields_map[field_name]
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^
KeyError: '<BoundField value=None errors=None>'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 333, in label_for_field
    field = _get_non_gfk_field(model._meta, name)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 302, in _get_non_gfk_field
    field = opts.get_field(name)
            ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/models/options.py", line 671, in get_field
    raise FieldDoesNotExist(
django.core.exceptions.FieldDoesNotExist: Table has no field named '<BoundField value=None errors=None>'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/templatetags/base.py", line 45, in render
    return super().render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/library.py", line 258, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/templatetags/admin_list.py", line 326, in result_list
    headers = list(result_headers(cl))
              ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/templatetags/admin_list.py", line 90, in result_headers
    text, attr = label_for_field(
                 ^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 361, in label_for_field
    raise AttributeError(message)
AttributeError: Unable to lookup '<BoundField value=None errors=None>' on Table or ClassificationAdmin
2023-11-01 18:51:43,133 (ERROR) - django.request - Internal Server Error: /admin/api/table/
Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/models/options.py", line 669, in get_field
    return self.fields_map[field_name]
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^
KeyError: '<BoundField value=None errors=None>'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 333, in label_for_field
    field = _get_non_gfk_field(model._meta, name)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 302, in _get_non_gfk_field
    field = opts.get_field(name)
            ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/models/options.py", line 671, in get_field
    raise FieldDoesNotExist(
django.core.exceptions.FieldDoesNotExist: Table has no field named '<BoundField value=None errors=None>'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/templatetags/base.py", line 45, in render
    return super().render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/library.py", line 258, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/templatetags/admin_list.py", line 326, in result_list
    headers = list(result_headers(cl))
              ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/templatetags/admin_list.py", line 90, in result_headers
    text, attr = label_for_field(
                 ^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 361, in label_for_field
    raise AttributeError(message)
AttributeError: Unable to lookup '<BoundField value=None errors=None>' on Table or ClassificationAdmin
2023-11-01 18:51:46,375 (ERROR) - django.request - Internal Server Error: /admin/api/table/
Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/models/options.py", line 669, in get_field
    return self.fields_map[field_name]
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^
KeyError: '<BoundField value=None errors=None>'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 333, in label_for_field
    field = _get_non_gfk_field(model._meta, name)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 302, in _get_non_gfk_field
    field = opts.get_field(name)
            ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/models/options.py", line 671, in get_field
    raise FieldDoesNotExist(
django.core.exceptions.FieldDoesNotExist: Table has no field named '<BoundField value=None errors=None>'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/templatetags/base.py", line 45, in render
    return super().render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/library.py", line 258, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/templatetags/admin_list.py", line 326, in result_list
    headers = list(result_headers(cl))
              ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/templatetags/admin_list.py", line 90, in result_headers
    text, attr = label_for_field(
                 ^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 361, in label_for_field
    raise AttributeError(message)
AttributeError: Unable to lookup '<BoundField value=None errors=None>' on Table or ClassificationAdmin
2023-11-01 18:57:46,372 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/admin.py changed, reloading.
2023-11-01 18:57:46,729 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-01 18:59:01,678 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/admin.py changed, reloading.
2023-11-01 18:59:02,070 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-01 18:59:06,021 (WARNING) - django.request - Not Found: /admin/api/table/
2023-11-01 18:59:16,124 (ERROR) - django.request - Internal Server Error: /admin/api/clasificacion/
Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/models/options.py", line 669, in get_field
    return self.fields_map[field_name]
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^
KeyError: 'group'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 333, in label_for_field
    field = _get_non_gfk_field(model._meta, name)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 302, in _get_non_gfk_field
    field = opts.get_field(name)
            ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/models/options.py", line 671, in get_field
    raise FieldDoesNotExist(
django.core.exceptions.FieldDoesNotExist: Clasificacion has no field named 'group'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/templatetags/base.py", line 45, in render
    return super().render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/library.py", line 258, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/templatetags/admin_list.py", line 326, in result_list
    headers = list(result_headers(cl))
              ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/templatetags/admin_list.py", line 90, in result_headers
    text, attr = label_for_field(
                 ^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 361, in label_for_field
    raise AttributeError(message)
AttributeError: Unable to lookup 'group' on Clasificacion or ClassificationAdmin
2023-11-01 19:01:25,668 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/admin.py changed, reloading.
2023-11-01 19:01:26,088 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-01 19:01:29,860 (ERROR) - django.request - Internal Server Error: /admin/api/clasificacion/
Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/options.py", line 686, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/utils/decorators.py", line 134, in _wrapped_view
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/views/decorators/cache.py", line 62, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/sites.py", line 242, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/utils/decorators.py", line 134, in _wrapped_view
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/options.py", line 1932, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/options.py", line 827, in get_changelist_instance
    list_display = self.get_list_display(request)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/pypro/djongofootball/api/admin.py", line 44, in get_list_display
    table_data = json.loads(last_instance.table)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.5/Frameworks/Python.framework/Versions/3.11/lib/python3.11/json/__init__.py", line 339, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not list
2023-11-01 19:03:07,359 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/admin.py changed, reloading.
2023-11-01 19:03:07,757 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-01 19:03:10,605 (ERROR) - django.request - Internal Server Error: /admin/api/clasificacion/
Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/options.py", line 686, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/utils/decorators.py", line 134, in _wrapped_view
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/views/decorators/cache.py", line 62, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/sites.py", line 242, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/utils/decorators.py", line 134, in _wrapped_view
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/options.py", line 1932, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/options.py", line 827, in get_changelist_instance
    list_display = self.get_list_display(request)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/pypro/djongofootball/api/admin.py", line 40, in get_list_display
    last_instance = Clasificacion.objects.all().last()
                    ^^^^^^^^^^^^^
NameError: name 'Clasificacion' is not defined
2023-11-01 19:03:26,648 (ERROR) - django.request - Internal Server Error: /admin/api/clasificacion/
Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/options.py", line 686, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/utils/decorators.py", line 134, in _wrapped_view
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/views/decorators/cache.py", line 62, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/sites.py", line 242, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/utils/decorators.py", line 134, in _wrapped_view
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/options.py", line 1932, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/options.py", line 827, in get_changelist_instance
    list_display = self.get_list_display(request)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/pypro/djongofootball/api/admin.py", line 40, in get_list_display
    last_instance = Clasificacion.objects.all().last()
                    ^^^^^^^^^^^^^
NameError: name 'Clasificacion' is not defined
2023-11-01 19:04:29,701 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/admin.py changed, reloading.
2023-11-01 19:04:30,097 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-01 19:04:33,410 (ERROR) - django.request - Internal Server Error: /admin/api/clasificacion/
Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/models/options.py", line 669, in get_field
    return self.fields_map[field_name]
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^
KeyError: 'group'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 333, in label_for_field
    field = _get_non_gfk_field(model._meta, name)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 302, in _get_non_gfk_field
    field = opts.get_field(name)
            ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/models/options.py", line 671, in get_field
    raise FieldDoesNotExist(
django.core.exceptions.FieldDoesNotExist: Clasificacion has no field named 'group'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/templatetags/base.py", line 45, in render
    return super().render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/template/library.py", line 258, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/templatetags/admin_list.py", line 326, in result_list
    headers = list(result_headers(cl))
              ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/templatetags/admin_list.py", line 90, in result_headers
    text, attr = label_for_field(
                 ^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/utils.py", line 361, in label_for_field
    raise AttributeError(message)
AttributeError: Unable to lookup 'group' on Clasificacion or ClassificationAdmin
2023-11-01 19:17:58,912 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/models.py changed, reloading.
2023-11-01 19:17:59,267 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-01 19:24:08,498 (WARNING) - djongo.sql2mongo.query - Not implemented alter command for SQL ALTER TABLE "api_legend" ADD CONSTRAINT "api_legend_pos_cd506833_pk" PRIMARY KEY ("pos")
2023-11-01 19:24:49,073 (WARNING) - djongo.sql2mongo.query - Not implemented alter command for SQL ALTER TABLE "api_legend" ADD CONSTRAINT "api_legend_pos_cd506833_pk" PRIMARY KEY ("pos")
2023-11-01 19:25:50,996 (WARNING) - djongo.sql2mongo.query - Not implemented alter command for SQL ALTER TABLE "api_legend" ADD CONSTRAINT "api_legend_pos_cd506833_pk" PRIMARY KEY ("pos")
2023-11-03 10:31:15,837 (WARNING) - djongo.sql2mongo.query - Not implemented alter command for SQL ALTER TABLE "api_table" ALTER COLUMN "type_id" TYPE string
2023-11-03 10:31:59,690 (WARNING) - djongo.sql2mongo.query - Not implemented alter command for SQL ALTER TABLE "api_table" ALTER COLUMN "type_id" TYPE string
2023-11-14 03:53:56,068 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-14 04:15:39,887 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/models.py changed, reloading.
2023-11-14 04:15:40,715 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-14 04:20:40,622 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/models.py changed, reloading.
2023-11-14 04:20:41,341 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-14 04:24:00,502 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/serializers.py changed, reloading.
2023-11-14 04:24:01,358 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-14 04:27:21,190 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/serializers.py changed, reloading.
2023-11-14 04:27:22,002 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-14 04:27:40,447 (WARNING) - djongo.sql2mongo.query - Not implemented alter command for SQL ALTER TABLE "api_jornada" ALTER COLUMN "local_goals" TYPE string
2023-11-14 04:30:48,972 (WARNING) - djongo.sql2mongo.query - Not implemented alter command for SQL ALTER TABLE "api_jornada" ALTER COLUMN "local_goals" TYPE string
2023-11-14 04:32:13,294 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/migrations/0001_initial.py changed, reloading.
2023-11-14 04:32:13,971 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-14 04:41:45,684 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/models.py changed, reloading.
2023-11-14 04:41:46,546 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-14 04:41:54,302 (WARNING) - djongo.sql2mongo.query - Not implemented alter command for SQL ALTER TABLE "api_jornada" ALTER COLUMN "date" TYPE string
2023-11-14 05:36:17,709 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/models.py changed, reloading.
2023-11-14 05:36:18,377 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-14 05:37:01,397 (WARNING) - django.request - Not Found: /
2023-11-14 05:37:01,803 (WARNING) - django.request - Not Found: /favicon.ico
2023-11-14 05:37:01,815 (WARNING) - django.request - Not Found: /apple-touch-icon-precomposed.png
2023-11-14 05:37:01,826 (WARNING) - django.request - Not Found: /apple-touch-icon.png
2023-11-14 05:38:31,585 (WARNING) - django.request - Not Found: /
2023-11-14 05:38:36,238 (WARNING) - django.request - Not Found: /api/proximos_partidos
2023-11-14 06:32:06,195 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/urls.py changed, reloading.
2023-11-14 06:32:06,574 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-14 06:32:17,278 (WARNING) - django.request - Not Found: /api/proximos_partidos/
2023-11-14 06:33:26,286 (WARNING) - django.request - Unauthorized: /api/proximos_partidos//partidos/
2023-11-14 06:38:19,284 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/urls.py changed, reloading.
2023-11-14 06:38:20,161 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-14 06:39:50,142 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/urls.py changed, reloading.
2023-11-14 06:39:50,523 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-14 06:40:46,626 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/urls.py changed, reloading.
2023-11-14 06:40:47,014 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-14 06:42:16,308 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-14 06:42:37,627 (WARNING) - djongo.sql2mongo.query - Not implemented alter command for SQL ALTER TABLE "api_jornada" ALTER COLUMN "date" TYPE string
2023-11-14 06:43:43,924 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-14 06:44:05,032 (WARNING) - django.request - Not Found: /
2023-11-14 06:44:10,936 (WARNING) - django.request - Not Found: /
2023-11-14 06:45:14,866 (WARNING) - django.request - Not Found: /
2023-11-14 06:45:20,331 (WARNING) - django.request - Not Found: /
2023-11-14 06:45:40,928 (WARNING) - django.request - Not Found: /api/jornada/
2023-11-14 06:46:29,234 (WARNING) - django.request - Not Found: /api/jornada/
2023-11-14 06:47:20,686 (WARNING) - django.request - Unauthorized: /api/jornada/partidos/
2023-11-14 06:47:31,475 (WARNING) - django.request - Unauthorized: /api/jornada/partidos/
2023-11-14 06:47:46,029 (WARNING) - django.request - Unauthorized: /api/jornada/partidos/
2023-11-14 06:49:33,747 (WARNING) - django.request - Not Found: /
2023-11-14 06:49:42,369 (WARNING) - django.request - Not Found: /
2023-11-14 06:49:52,344 (WARNING) - django.request - Method Not Allowed: /auth/jwt/create/
2023-11-14 06:58:59,487 (WARNING) - django.request - Unauthorized: /api/jornada/partidos/
2023-11-14 06:58:59,780 (WARNING) - django.request - Not Found: /favicon.ico
2023-11-14 07:03:07,702 (WARNING) - django.request - Unauthorized: /api/jornada/partidos/
2023-11-14 07:03:11,194 (WARNING) - django.request - Unauthorized: /api/jornada/partidos/
2023-11-14 07:05:35,249 (WARNING) - django.request - Unauthorized: /api/jornada/partidos/
2023-11-14 07:05:37,281 (WARNING) - django.request - Unauthorized: /api/jornada/partidos/
2023-11-14 07:05:40,142 (WARNING) - django.request - Unauthorized: /api/jornada/partidos/
2023-11-14 07:06:47,337 (WARNING) - django.request - Unauthorized: /api/jornada/partidos/
2023-11-14 07:07:21,667 (WARNING) - django.request - Unauthorized: /api/jornada/partidos/
2023-11-14 07:07:23,176 (WARNING) - django.request - Unauthorized: /api/jornada/partidos/
2023-11-14 07:08:01,054 (WARNING) - django.request - Unauthorized: /api/jornada/partidos/
2023-11-14 07:09:13,341 (WARNING) - django.request - Unauthorized: /api/jornada/partidos/
2023-11-14 07:10:59,982 (WARNING) - django.request - Unauthorized: /api/jornada/partidos/
2023-11-14 07:11:02,297 (WARNING) - django.request - Unauthorized: /api/jornada/partidos/
2023-11-14 07:11:31,446 (WARNING) - django.request - Not Found: /
2023-11-14 07:13:12,389 (ERROR) - django.request - Internal Server Error: /admin/core/user/add/
Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/djongo/sql2mongo/query.py", line 808, in __iter__
    yield from iter(self._query)
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/djongo/sql2mongo/query.py", line 167, in __iter__
    yield self._align_results(doc)
          ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/djongo/sql2mongo/query.py", line 269, in _align_results
    if selected.table == self.left_table:
       ^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/djongo/sql2mongo/sql_tokens.py", line 133, in table
    return alias2token[name].table
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/djongo/sql2mongo/sql_tokens.py", line 133, in table
    return alias2token[name].table
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/djongo/sql2mongo/sql_tokens.py", line 133, in table
    return alias2token[name].table
           ^^^^^^^^^^^^^^^^^^^^^^^
  [Previous line repeated 912 more times]
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/djongo/sql2mongo/sql_tokens.py", line 130, in table
    name = self.given_table
           ^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/djongo/sql2mongo/sql_tokens.py", line 141, in given_table
    name = self._token.get_real_name()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/sqlparse/sql.py", line 361, in get_real_name
    return self._get_first_name(dot_idx)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/sqlparse/sql.py", line 386, in _get_first_name
    return token.get_name()
           ^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/sqlparse/sql.py", line 355, in get_name
    return self.get_alias() or self.get_real_name()
           ^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/sqlparse/sql.py", line 344, in get_alias
    _, ws = self.token_next_by(t=T.Whitespace)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/sqlparse/sql.py", line 244, in token_next_by
    return self._token_matching(funcs, idx, end)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/sqlparse/sql.py", line 223, in _token_matching
    if func(token):
       ^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/sqlparse/sql.py", line 242, in <lambda>
    funcs = lambda tk: imt(tk, i, m, t)
                       ^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/sqlparse/utils.py", line 100, in imt
    elif types and any(token.ttype in ttype for ttype in types):
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/sqlparse/utils.py", line 100, in <genexpr>
    elif types and any(token.ttype in ttype for ttype in types):
                       ^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/sqlparse/tokens.py", line 19, in __contains__
    return item is not None and (self is item or item[:len(self)] == self)
                                                 ^^^^^^^^^^^^^^^^^^^^^^^^
RecursionError: maximum recursion depth exceeded in comparison

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/djongo/cursor.py", line 76, in fetchone
    return self.result.next()
           ^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/djongo/sql2mongo/query.py", line 797, in __next__
    result = next(self._result_generator)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/djongo/sql2mongo/query.py", line 830, in __iter__
    raise exe from e
djongo.exceptions.SQLDecodeError: 

	Keyword: FAILED SQL: SELECT %(0)s AS "a" FROM "core_user" WHERE "core_user"."username" = %(1)s LIMIT 1
Params: (1, 'ric')
Version: 1.3.6
	Sub SQL: None
	FAILED SQL: None
	Params: None
	Version: None

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/utils.py", line 98, in inner
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/djongo/cursor.py", line 81, in fetchone
    raise db_exe from e
djongo.database.DatabaseError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/options.py", line 686, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/utils/decorators.py", line 134, in _wrapped_view
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/views/decorators/cache.py", line 62, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/sites.py", line 242, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/views/decorators/debug.py", line 92, in sensitive_post_parameters_wrapper
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/utils/decorators.py", line 134, in _wrapped_view
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/auth/admin.py", line 119, in add_view
    return self._add_view(request, form_url, extra_context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/auth/admin.py", line 147, in _add_view
    return super().add_view(request, form_url, extra_context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/options.py", line 1891, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/utils/decorators.py", line 134, in _wrapped_view
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/options.py", line 1751, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/admin/options.py", line 1797, in _changeform_view
    form_validated = form.is_valid()
                     ^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/forms/forms.py", line 205, in is_valid
    return self.is_bound and not self.errors
                                 ^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/forms/forms.py", line 200, in errors
    self.full_clean()
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/forms/forms.py", line 439, in full_clean
    self._post_clean()
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/contrib/auth/forms.py", line 129, in _post_clean
    super()._post_clean()
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/forms/models.py", line 498, in _post_clean
    self.validate_unique()
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/forms/models.py", line 507, in validate_unique
    self.instance.validate_unique(exclude=exclude)
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/models/base.py", line 1207, in validate_unique
    errors = self._perform_unique_checks(unique_checks)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/models/base.py", line 1317, in _perform_unique_checks
    if qs.exists():
       ^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/models/query.py", line 1226, in exists
    return self.query.has_results(using=self.db)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/models/sql/query.py", line 592, in has_results
    return compiler.has_results()
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1366, in has_results
    return bool(self.execute_sql(SINGLE))
                ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1409, in execute_sql
    val = cursor.fetchone()
          ^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/utils.py", line 97, in inner
    with self:
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/django/db/utils.py", line 98, in inner
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ric/.local/share/virtualenvs/djongofootball-2Ah-EyYR/lib/python3.11/site-packages/djongo/cursor.py", line 81, in fetchone
    raise db_exe from e
django.db.utils.DatabaseError
2023-11-14 07:17:28,454 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/views.py changed, reloading.
2023-11-14 07:17:28,892 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-14 07:25:54,998 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/serializers.py changed, reloading.
2023-11-14 07:25:55,437 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-14 07:26:18,271 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/serializers.py changed, reloading.
2023-11-14 07:26:19,126 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-14 07:27:00,389 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/views.py changed, reloading.
2023-11-14 07:27:01,244 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-14 07:40:29,642 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/migrations/0002_initial.py changed, reloading.
2023-11-14 07:40:29,931 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-14 07:40:52,729 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/football_quiz/settings.py changed, reloading.
2023-11-14 07:40:53,545 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-14 16:23:47,608 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/models.py changed, reloading.
2023-11-14 16:23:48,459 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-14 16:23:49,705 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/serializers.py changed, reloading.
2023-11-14 16:23:50,354 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-15 12:52:31,491 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/football_quiz/settings.py changed, reloading.
2023-11-15 12:52:32,354 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-15 12:55:44,118 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/models.py changed, reloading.
2023-11-15 12:55:44,955 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-15 14:40:46,360 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/serializers.py changed, reloading.
2023-11-15 14:40:47,184 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-15 14:40:49,352 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/models.py changed, reloading.
2023-11-15 14:40:49,565 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-15 14:42:54,133 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/serializers.py changed, reloading.
2023-11-15 14:42:54,543 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-15 14:43:09,123 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/serializers.py changed, reloading.
2023-11-15 14:43:09,484 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
2023-11-15 14:43:37,476 (INFO) - django.utils.autoreload - /Users/ric/pypro/djongofootball/api/models.py changed, reloading.
2023-11-15 14:43:38,178 (INFO) - django.utils.autoreload - Watching for file changes with StatReloader
